Tasks/AzureFileCopy/task.json

```diff
@@ -1,4 +1,4 @@
-﻿{
```

Comment:
@TingluoHuang Wouldn't it be more appropriate if the task handler infrastructure discards the BOM? Otherwise customers will likely run into the same frustrating BOM issue, correct?